### DISCUSSION TITLE
Нельзя добавлять больше 6 позиций

### DISCUSSION THREAD

**Discussion by Sidovskiy** [cdek-it/wordpress#42] [Q&A] [answered]
Всем привет. Добавляю 6 различных товаров на сайте и пишет  предупреждение...

Нельзя добавлять больше 6 позиций
Уберите лишние позиции из корзины – остальные можно добавить в другом заказе.

А если у меня товар брелки для ключей?

**Comment by vermorag** [comment, marked as answer]
Добрый день. Наш плагин не ограничивает количество позиций, которые возможны в заказе.
